apps/website/src/pages/docs/carousel.mdx

```diff
@@ -14,40 +14,40 @@ import * as AllExamples from 'examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core--carousel' />
-
 <LiveExample src='Carousel.main.tsx'>
   <AllExamples.CarouselMainExample client:load />
 </LiveExample>
 
 The Carousel component consists of a set of slides, normally displayed one at a time. A navigation section is
-located below the slides, consisting of "dots" and "previous"/"next" buttons, used for changing slides.
+located below the slides, consisting of "dots" ([Carousel.DotsList](#dots-list)) and "previous"/"next" ([Carousel.Navigation](#navigation)) buttons, used for changing slides.
 
 The currently shown slide can also be changed using the left/right arrow keys or by dragging on a touch device.
 
 ## Usage
 
-Carousels can be controlled externally by using the properties `activeSlideIndex` and `onSlideChange`.
+### Controlled State
+
+By default, the carousel maintains its own state. But it can also be controlled externally using `activeSlideIndex` and `onSlideChange`.
 
 <LiveExample src='Carousel.controlled.tsx'>
   <AllExamples.CarouselControlledExample client:load />
 </LiveExample>
 
-The subcomponents can be used outside of Carousel. The following example shows how `Carousel.DotsList` can be used without the `Carousel` component. Clicking on the right half of the slide will advance the carousel to the next slide, whereas clicking on the left half will go to the previous slide.
+### Only Dots
+
+`Carousel.DotsList` can be used without the `Carousel` component. Here is an example to demonstrate this where clicking on the right/left half of the slide will go one slide forward/backward.
 
 <LiveExample src='Carousel.onlyDots.tsx'>
   <AllExamples.CarouselOnlyDotsExample client:load />
 </LiveExample>
 
 ## Props
 
-Properties table is to be defined.
-
 <PropsTable path={frontmatter.propsPath} />
 
 ## Subcomponents
 
-This component uses a composition approach so it should be used with the provided subcomponents.
+This component uses a composition approach, similar to most of the other components.
```

Comment:
Should we explain that not all subcomponents are required to be used? When designing the carousel api, we took extra care to reduce verbosity

I'm also thinking the "Usage" heading would be a better spot for such explanation

Reply:
added where? i don't see it.

could you try to make smaller, more frequent commits and include links to them in your responses so that it's easier to review?